e2e-tests/gef/cypress/support/commands/loadData.js

```diff
@@ -2,14 +2,11 @@
  * Mock data loader:
  * 1. Clears all the collections
  * 2. Inserts mocked data
-*/
+ */
 
 const loadData = () => {
-  // Ensure project root directory
-  cy.exec('if [ -d "dtfs2" ]; then cd dtfs2; fi');
-
   // Load mock data
-  cy.exec('npm run load');
+  cy.exec('cd ../.. && npm run load');
```

Comment:
Now we have `package.json` in `e2e-tests` should we not add `npm run load` script which then refers to the `root` package `load` script and then we can then directory traversing?